src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -96,34 +96,61 @@ private void LoadingStatusIndicator_PropertyChanged(object sender, PropertyChang
             });
         }
 
-        // Indicates wether check boxes are enabled on packages
-        private bool _checkBoxesEnabled;
+        public bool CheckBoxesEnabled { get; set; }
 
-        public bool CheckBoxesEnabled
+        public bool IsSolution { get; set; }
+
+        public ObservableCollection<object> Items { get; } = new ObservableCollection<object>();
+        private ICollectionView CollectionView
         {
             get
             {
-                return _checkBoxesEnabled;
+                return CollectionViewSource.GetDefaultView(Items);
             }
-            set
-            {
-                _checkBoxesEnabled = value;
+        }
 
-                if (!_checkBoxesEnabled)
+        /// <summary>
+        /// Count of Items (excluding Loading indicator) that are currently shown after applying any UI filtering.
+        /// </summary>
+        private int FilterCount
+        {
+            get
+            {
+                if (CollectionView.Filter != null)
                 {
-                    // the current tab is not "updates", so the container
-                    // should become invisible.
-                    _updateButtonContainer.Visibility = Visibility.Collapsed;
+                    var listCollectionView = (ListCollectionView)CollectionView;
+                    return listCollectionView.OfType<PackageItemListViewModel>().Count();
+                }
+                else
+                {
+                    return Items.OfType<PackageItemListViewModel>().Count();
```

Comment:
Same here, use `.Count`.

Reply:
See other explanation

Reply:
Items is ICollectionView , so it's only IEnumerable, not sure how I found the Count prop in the docs last night. 
Regardless, resolving.